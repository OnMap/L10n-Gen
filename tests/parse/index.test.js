const { parseSourceFilesToKeys } = require('../../src/parse/index.js');

describe('iOS parse', () => {
  it('source files to keys', async () => {
    const result = await parseSourceFilesToKeys('./tests/mocks_data', 'IOS');
    expect(result).toMatchSnapshot();
  });
});
