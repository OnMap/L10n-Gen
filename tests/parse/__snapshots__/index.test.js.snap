// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`iOS parse source files to keys 1`] = `Array []`;
